File: Java/Problem1.java
class Solution {
    public double findMedianSortedArrays(int[] nums1, int[] nums2) {
        List<Integer> merge = new ArrayList<>();
        int i = 0, j = 0;

        while (i < nums1.length || j < nums2.length) {
            if (j >= nums2.length || (i < nums1.length && nums1[i] <= nums2[j])) {
                merge.add(nums1[i++]);
            } else {
                merge.add(nums2[j++]);
            }
        }

        int n = merge.size();
        if (n % 2 == 0) {
            return (merge.get(n / 2 - 1) + merge.get(n / 2)) / 2.0;
        } else {
            return merge.get(n / 2);
        }
    }
}

File: Python/Problem1.py
class Solution(object):
    def findMedianSortedArrays(self, nums1, nums2):
        merge = []
        i=0
        j=0
        while i < len(nums1) and j < len(nums2):
            if nums1[i] <= nums2[j]:
                merge.append(nums1[i])
                i += 1
            else:
                merge.append(nums2[j])
                j += 1
        merge.extend(nums1[i:])
        merge.extend(nums2[j:])
        n=len(merge)
        if n % 2 == 0:
            return (merge[n // 2 - 1] + merge[n // 2]) / 2.0
        else:
            return merge[n // 2]